src/components/stats/StatsGraph.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
```

Comment:
Removed the casing after [1a09522](https://github.com/osmlab/to-fix/pull/160/commits/1a095220eb4c62e3458c09a8ce0abe4b67428bef), [7af5d9c](https://github.com/osmlab/to-fix/pull/160/commits/7af5d9c3c14726bdef4e450d76128dc007364e6f) and [bf8c7da](https://github.com/osmlab/to-fix/pull/160/commits/bf8c7dafdd432e9f772e67dde677c03686cb1eac).